lib/central_mail/datestamp_pdf.rb

```diff
@@ -12,16 +12,18 @@ def initialize(file_path, append_to_stamp: nil)
 
     def run(settings)
       stamp_path = Common::FileHelpers.random_file_path
-      generate_stamp(stamp_path, settings[:text], settings[:x], settings[:y], settings[:text_only], settings[:size])
+      generate_stamp(stamp_path, settings[:text], settings[:x], settings[:y], settings[:text_only], settings[:size],
+                     settings[:form_submission_date])
```

Comment:
See my comment below (https://github.com/department-of-veterans-affairs/vets-api/pull/14900/files#r1434240037)

Reply:
@tommasina-va I read your note and see that the behavior will continue to be the same unless the optional `form_submission_date` passed in, which seems good to me.

Can I make one suggestion? What do you think about naming the optional setting `timestamp` instead of `form_submission_date`? This would allow it to be more flexible for any team that wants to use a timestamp other than `Time.zone.now`. Then you can still pass the form submission date in your usages of the class, but other teams can pass any custom date. I think this also makes the purpose of this particular setting clearer when just reading the settings list.